src/state/backendNetworks/backendNetworks.ts

```diff
@@ -9,6 +9,7 @@ import { IS_TEST } from '@/env';
 import { BackendNetwork, BackendNetworkServices, chainHardhat, chainHardhatOptimism, ChainId } from '@/state/backendNetworks/types';
 import { GasSpeed } from '@/__swaps__/types/gas';
 import { useConnectedToHardhatStore } from '@/state/connectedToHardhat';
+import { colors as globalColors } from '@/styles';
```

Comment:
Small and unimportant, but given there is an export from the @/design-system called globalColors, but you do have to rename colors to something else here, something like themeColors might be less confusing